iamproperty/iamproperty.github.io
# Ds 51/property searchbar

## Summary of Changes:
1. Vue component
2. New CSS module
3. JS form helpers

## Rationale
A searchbar component designed with properties in mind.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/property-searchbar

## Ticket(s)
DS-51

## Checklist

## Notes
